VTEX - Shopper Data API.json

```diff
@@ -0,0 +1,718 @@
+                "tags": ["Shopper documents"],
+                "parameters": [
+                    {
+                        "name": "_schema",
+                        "in": "query",
+                        "required": true,
+                        "schema": {
+                            "type": "string",
+                            "default": "v1"
+                        },
+                        "description": "Schema name. The value must be `v1`."
+                    },
+                    {
+                        "$ref": "#/components/parameters/Content-Type"
+                    },
+                    {
+                        "$ref": "#/components/parameters/Accept"
+                    }
+                ],
+                "requestBody": {
+                    "required": true,
+                    "content": {
+                        "application/json": {
+                            "schema": {
+                                "$ref": "#/components/schemas/ShopperDocumentRequest"
+                            },
+                            "example": {
+                                "userId": "8abea412-4702-46fb-b835-3edb29d8a261",
+                                "email": "john@vtex.com",
+                                "firstName": "John",
+                                "lastName": "Doe",
+                                "document": "514.427.690-33",
+                                "documentType": "cpf",
+                                "cards": [],
+                                "phone": "11917118990",
+                                "phone2": "11987654321",
+                                "marketingOptIn": true
+                            }
+                        }
+                    }
+                },
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "$ref": "#/components/schemas/CreateDocumentResponse"
+                                },
+                                "example": {
+                                    "Id": "shopper-a26301e6-1a1e-11f0-b37f-e46d3d0578c6",
+                                    "Href": "",
+                                    "DocumentId": "a26301e6-1a1e-11f0-b37f-e46d3d0578c6"
+                                }
+                            }
+                        }
+                    }
+                },
+                "security": [
+                    {
+                        "VtexIdclientAutCookie": []
+                    }
+                ]
+            }
+        },
+        "/api/dataentities/shopper/documents/{userId}": {
```

Comment:
```suggestion
        "/api/dataentities/shopper/documents/{shopperId}": {
```

Reply:
"User ID" can lead users to believe this is the ID from [Create storefront user with username](https://developers.vtex.com/docs/api-reference/vtex-id-api#post-/api/authenticator/storefront/users), but I think, from mdv2 standard mechanics, this path param should be the document ID created in the POST above.